### PR TITLE
Allow `--github` flag on `cargo add`

### PR DESCRIPTION
**Pull request by johantell**
This will ask for a github username and then fetch
all necessary information needed to add the user to the
coauthors list

Co-Authored-By: Louice Danielsson <21656089+louiced@users.noreply.github.com>

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -2,28 +2,43 @@ mod cli_input;
...
+use input_command::{InputCommand, UserType};
 use std::env;
 
 fn main() -> Result<(), ExitFailure> {
     let args: Vec<String> = env::args().collect();
 
+    // let username = "louiced";
+    // let response = github_api::get_coauthor_from_username(username.to_string());
+    // println!("{:#?}", response);
+
```
Comment (louiced) [inline src/main.rs]: We might want to remove this though? ;)

File: src/main.rs [hunk 2]
```diff
@@ -101,6 +116,9 @@ fn print_help_section() {
 
     USAGE:
       add                               Starts a prompt to add an coauthor.
+
```
Comment (louiced) [inline src/main.rs]: I think it would look better without this newline.

File: src/main.rs (hunk 1 above)
Comment (johantell) [reply src/main.rs]: Ohh yes we do :D

File: src/main.rs (hunk 2 above)
Comment (johantell) [reply src/main.rs]: I was afraid that the text from the section above would get mixed up. Maybe not a problem. I'll change

**Comment by louiced** [comment]
🎉

**Pull request merged**